README.md

```diff
@@ -1,32 +1,32 @@
 This here's a tail for all the fellas
-Tryin; to do what those ladies tell us
+Trying to do what those ladies tell us
 Get shot down cause ya over-zealous
 Play hard to get females get jealous
 Okay smarty go to a party
 Girls are scantily clad and showin; body
 A chick walks by you wish you could sex her
 But you're standing on the wall like you was Poindexter
 Next days function high class luncheon
-Food is served and you're stone-cold munchin'
+Food is served and you're stone-cold munching
 Music comes on people start to dance
 But then you ate so much you nearly split your pants
 A girl starts walking guys start gawking
 Sits down next to you and starts talking
 Says she wants to dance cause she likes to groove
 So come on fatso and just bust a move
 
-You're on a mission and your wishin'
+You're on a mission and your wishing
```

Comment:
Maybe keep "wishin'" because it rhymes with "mission".
There are some other internal rhymes like:

* Okay **smarty** go to a **party**
* A girl starts **walking** guys start **gawking**